packages/cerebral-provider-firebase/src/deleteUser.js

```diff
@@ -0,0 +1,9 @@
+import firebase from 'firebase'
+
+export default function deleteUser (password) {
+  return new Promise((resolve, reject) => {
+    const user = firebase.auth().currentUser
+
+    return user.delete().then(() => resolve(), (error) => reject({error}))
```

Comment:
Can user be null here? According to docs "You can also get the currently signed-in user by using the currentUser property. If a user isn't signed in, currentUser is null:"